src/tasks/checkEnvironment.js

```diff
@@ -29,14 +31,11 @@ var self = {
       return false;
     }
 
-    // check for windows or Mac OSX
-    if (['win32', 'darwin'].indexOf(process.platform) > -1) {
-      return self.boot2dockerExists();
-    }
```

Comment:
If `dockerExists` no longer checks for dockerMachine and/or boot2docker, the behavior of https://github.com/artificialio/sane/blob/master/src/commands/new.js#L221 will be changed.  Is that intended?